src/components/cc-global-header.vue

```diff
@@ -128,11 +128,42 @@ export default defineComponent({
       ];
     } else {
       vm.menuLoading = true;
+      axios.interceptors.response.use(
+        undefined,
+        function retryApiRequest(error) {
+          var config = error.config;
+          // If config does not exist or the retry option is not set, reject
+          if (!config || !config.retry) return Promise.reject(error);
+
+          // Set the variable for keeping track of the retry count
+          config.retryCount = config.retryCount || 0;
+
+          // Check if we've maxed out the total number of retries
+          if (config.retryCount >= config.retry) {
+            // Reject with the errors
+            return Promise.reject(error);
+          }
+
+          // Increase the retry count
+          config.retryCount += 1;
+
+          // Create new promise for exponential backoff
+          var exponentialBackOff = new Promise(function (resolve) {
+            setTimeout(function () {
+              resolve();
+            }, config.retryDelay || 1000);
```

Comment:
Let's define an explanatory variable to use here.
```suggestion
            }, config.retryDelay || defaultRetryDelay);
```